HACKING.md

```diff
@@ -1,288 +1,169 @@
 
-Install these packages in your Goldstone virtualenv:
+Expand the "Advanced" section of the Network settings for the boot2docker-vm, and click on the "Port Forwarding" button.  Then configure forwarders as described in the table below.
+|Name|Protocol|Host IP|Host Port|Guest IP|Guest Port|
+|----|--------|-------|---------|--------|----------|
+|es_9200_RDO|TCP|172.24.4.1|9200||9200|
+|es_9200_local|TCP|127.0.0.1|9200||9200|
+|es_9300_RDO|TCP|172.24.4.1|9300||9300|
+|es_9300_local|TCP|127.0.0.1|9300||9300|
+|logstash_syslog_RDO|TCP|172.24.4.1|5514||5514|
+|logstash_syslog_local|TCP|127.0.0.1|5514||5514|
+|logstash_metrics_RDO|UDP|172.24.4.1|5516||5516|
+|logstash_metrics_local|UDP|127.0.0.1|5516||5516|
+|postgres_RDO|TCP|172.24.4.1|5432||5432|
+|postgres_local|TCP|127.0.0.1|5432||5432|
+|redis_local|TCP|127.0.0.1|6379||6379|
+|ssh|TCP|127.0.0.1|2022||22|
 
-```bash
-$ workon goldstone-server
-$ brew install elasticsearch phantomjs redis postgresql
-```
 
-### Set up your databases
-Create your development and test databases, and create a "goldstone" database user:
-    
-```bash
-$ createdb goldstone_dev
-$ createdb goldstone_test
-$ createuser goldstone -d
-```
+![enter image description here](https://lh3.googleusercontent.com/Hy1sDfWbYbLvhJjZa7kNSXXImGtri7zIlwPEazNwk3s=w797-h634-no)
 
-Edit your `pg_hba.conf` file.  If you installed with brew, this should be in `/usr/local/var/postgres/`.  See the [INSTALL file](INSTALL.md) for the modifications.
+Activate the Virtualenv 
+----------------------------------------------------
+Once all of the initial setup has been completed, you can activate the virtualenv by running the workon command. 
 
-After you've edited pg_hba.conf, reload postgres:
+> $ workon goldstone-server
 
-```bash
-$ pg_ctl reload
-```
-
-If you want remote access to postgres, you will also need to add an entry to
-`/etc/sysconfig/iptables` like:
-
-    -A INPUT -p tcp -m state --state NEW -m tcp --dport 5432 -m comment --comment "postgres incoming" -j ACCEPT 
+This command will start the required VMs, docker containers, and celery processes.  Running `deactivate` will stop everything.  
 
-And edit `/usr/local/var/postgres/postgresql.conf` to configure it to listen on 
-external addresses::
+The first time you enter the virtualenv, you should also install the project requirements, and some additional utilities.
 
-    listen_address='*'
-
-Then restart iptables:
+    $ pip install --upgrade pip
+    $ pip install -r requirements.txt
+    $ pip install -r test-requirements.txt
+    $ pip install flower
 
-```bash
-$ service iptables restart
-```
+If the requirements files change, you should rerun the `pip install` commands.
 
-### Get the code
+*Note that the goldstone-server virtualenv is only meant to be run in a single terminal window. *
```

Comment:
This isn't bolded or emphasized in the output. Two **? Some other formatting problem?